apm-agent-plugins/apm-mongodb/apm-mongodb3-plugin/src/main/java/co/elastic/apm/agent/mongodb/v3/ConnectionInstrumentation.java

```diff
@@ -88,7 +88,7 @@ public static Object onEnter(@Advice.This Connection thiz,
                 namespace.getDatabaseName(),
                 namespace.getCollectionName(),
                 command,
-                address.getHost(), address.getPort());
+                address.getHost(), address.getPort(), null);
```

Comment:
[for reviewer] this is the case where the statement can't be captured with some 3.x drivers.